cms-plugin/src/main/java/com/agiletec/plugins/jacms/aps/system/services/content/model/attribute/LinkAttribute.java

```diff
@@ -66,34 +70,38 @@ public Object getAttributePrototype() {
     @Override
     public Element getJDOMElement() {
         Element attributeElement = this.createRootElement("attribute");
-        if (null != this.getSymbolicLink()) {
+        Iterator<String> langIter = this.getSymbolicLinks().keySet().iterator();
+        while (langIter.hasNext()) {
+            String currentLangCode = langIter.next();
```

Comment:
Can we use a simple foreach instead of iterator?

Reply:
Fixed